plugins/vppv2/model/interfaces/interface.proto

```diff
@@ -106,3 +107,35 @@ message TapLink {
     uint32 rx_ring_size = 4;    /* Rx ring buffer size; must be power of 2; default is 256; only for TAP v.2 */
     uint32 tx_ring_size = 5;    /* Tx ring buffer size; must be power of 2; default is 256; only for TAP v.2 */
 }
+
+message IPSecLink {
+    bool esn = 2;                       /* Extended sequence number */
+    bool anti_replay = 3;               /* Anti replay option */
+    string local_ip = 4;                /* Local IP address */
+    string remote_ip = 5;               /* Remote IP address */
+    uint32 local_spi = 6;               /* Local security parameter index */
+    uint32 remote_spi = 7;              /* Remote security parameter index */
+
+    enum CryptoAlg {
```

Comment:
These fields + IntegAlg are currently duplicated here and in ipsec.proto. I can pull them out and import from ipsec, but maybe @ondrej-fabry you can refactor it together with the rest of the NB API as a part of the #1022